source/newcommitter.md

```diff
@@ -295,11 +295,11 @@ This is the followup email after the new committer has accepted the invitation
 
     Just as before you became a committer, participation in any ASF community
     requires adherence to the ASF Code of Conduct:
-      https://www.apache.org/foundation/policies/conduct.html
+      https://www.apache.org/foundation/policies/conduct.html.
     
     [PROJECT should insert its own guidelines here; if none are available,
-     the Apache Forrest guidelines are available as a template.]
-      https://forrest.apache.org/guidelines.html
+     the Apache Forrest guidelines are available as a template.
+      https://forrest.apache.org/guidelines.html.]
```

Comment:
Forrest is now in the attic and might not be a great example any more...

Reply:
Good point @clr-apache
I have created Jira ticket for it https://issues.apache.org/jira/browse/COMDEV-421
So, it will not block us from merging the all other suggested changes which is incorporated #62